pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -272,6 +293,10 @@ type XdsConnection struct {
 	// is added to the map of active.
 	added bool
 
+	// vhdsEnabled will be true if vhds is enabled.  If that's the case, then virtual hosts will be sent over VHDS and
+	// the list from RDS will be empty. If false, RDS will send virtual hosts
```

Comment:
Can't we send an initial list, and get vhds for not-found ? Any other way we can 'warm' up a set of known vhosts, and only use vhds for infrequent ones  ?

Reply:
just looking this old PR - if ever we do this, we should specify vhosts and vhds both. So the initial vhosts are delivered and vhds is only delivered for later vhosts